df_py/util/dftool_module.py

```diff
@@ -54,7 +65,7 @@
   dftool vebals ST FIN NSAMP CSV_DIR CHAINID [RETRIES]
   dftool challenge_data CSV_DIR [DEADLINE] [RETRIES]
   dftool predictoor_data CSV_DIR START_DATE END_DATE CHAINID [RETRIES]
-  dftool calc CSV_DIR TOT_OCEAN [START_DATE] [IGNORED] - from stakes/etc csvs, output rewards csvs across Volume + Challenge + Predictoor DF
+  dftool calc CSV_DIR TOT_OCEAN START_DATE [SUBSTREAM_NAME] [IGNORED] - from stakes/etc csvs, output rewards csvs across Volume + Challenge + Predictoor DF
```

Comment:
I really like you you put aggregation of csv rewards across substreams into `dftool dispense_active`. Makes lots of sense, and the implementation looks good.

----

Now that I see this implementation, I realize it's much cleaner to have a _different_ dftool command for each substream. 

I propose:
```
dftool calc_volume_rewards ...
dftool calc_challenge_rewards ...
dftool calc_predictoor_rewards ...
```

(I see that Calina had a similar comment. Good! :)

----

Small thing: you can remove the "IGNORED" stuff. That's only there because I didn't want to cause downstream breakage in df-sql. But you're making enough changes that you have to change/test downstream anyway. Perhaps as you make the changes above, then "IGNORED" will naturally get tossed out.

Reply:
I did have a similar comment, and the underlying functionality implemented by Berkay looks great!

But in regards to having different commands, I think would better with a parameter. i.e. `dftool calc_rewards volume...` not, `dftool calc_volume_rewards...` because it makes the handling really ugly and if we can have a parameter instead of a new function it seems to make more sense.

Reply:
Fixed. I am marking this as resolved, hope I'm not overstepping.